src/index.js

```diff
@@ -2,18 +2,61 @@ import "./styles/normalize.css";
 import "./styles/styles.css";
 import { sortArrayByName } from './utils';
 import { moviesData } from './movies';
-import { getRoot, getMovieWrap } from './accessors';
-import { renderWelcomeLetter, renderMovies } from './render';
-
+import { getMovieWrap, getMovieContainerDiv, getMovieDetailsContainerDiv } from './accessors';
+import { renderMovies, renderMovieDetails, clearMovies } from './render';
 
 function init() {
-    const root = getRoot();
-    renderWelcomeLetter(root);
-    console.log(moviesData);
 
     const renderedMovies = renderMovies(moviesData);
+    const movieContainerDiv = getMovieContainerDiv();
+    movieContainerDiv.append(renderedMovies);
+
+
     const movieWrap = getMovieWrap();
-    movieWrap.append(renderedMovies);
+    movieContainerDiv.addEventListener('click', (evt) => {
```

Comment:
as the first step for future refactor, I can suggest moving event handlers to separate named functions